src/qt/textentry.cpp

```diff
@@ -104,11 +105,20 @@ wxString wxTextEntry::DoGetValue() const
     return wxString();
 }
 
-void wxTextEntry::DoSetValue(const wxString &WXUNUSED(value), int WXUNUSED(flags))
+void wxTextEntry::DoSetValue(const wxString& value, int flags)
 {
+    wxTextEntryBase::DoSetValue(value, flags);
 }
```

Comment:
Maybe we should just stop overriding it? Or do you plan to do something else than calling the base class version here in the (near) future?

Reply:
No I don't have any plan about this function. at least for now. So feel free to remove this override when you merge this PR. Thanks!